### PR TITLE
varnishhist improvements: backend, refresh, timebend, y scale

### PR DESCRIPTION
**Pull request by nigoroll**
Various improvements

### CODE REVIEW

File: bin/varnishhist/varnishhist.c [hunk 1]
```diff
@@ -440,6 +448,13 @@ main(int argc, char **argv)
 		case 'h':
 			/* Usage help */
 			usage(0);
+		case 'p':
+			delay = strtod(optarg, NULL);
+			if (delay == 0) {
```
Comment (Dridi) [inline bin/varnishhist/varnishhist.c]: Check `delay <= 0` instead?

**Comment by nigoroll** [comment]
bugwash:
* ok'ed by @mbgrydeland 
* good idea by @Dridi : move profiles to tbl and generate rst

**Comment by nigoroll** [comment]
fully merged after rebase

**Comment by lkarsten** [comment]
Backport review: let's get this into 4.1 as well.

**Comment by nigoroll** [comment]
hi @hermunn 
* 149b6287dbc21141b90f7b9afd3132d26d3fb8c9
* 39181b71c29e057958bd8bf83f6e904d30ab80a5
* 2fef611dcd2e3c8d8f582b538a02b1d4e6c9a249
* 95793d1061f71d70d1934c8bc18a260f0eec9082
* 0e9928d8f45d0cc9efd6e00b241d117d9f872bed
* 12fe87bc6d1567ba31af31fe3b3669156ecd4aa2
* 1f8d7e1c73c0bac87ef2b84d2cd1ecfe7f171e2b
* 9948f96425e7c56a92f1478ef89eb1ac932d422d
* 60cbe5df163bb5d3d9765614a170ab8433789d95
* 1ce9cfc64fe50c3ba9c471ccebfa88f53d81cd62
* 7b365f36941427b1922538dc91a6da888ee81ef0
* 4ef8289279aae89354674fc2fb1c746c4e8b4fe4
* 6a93c366d840b2de278c882f4afc9e69c722789f
* 0a8d391520b30218f6adc29b6447c32771d76343
* b99c468651251b6b0b6d0381c9d34799521310b6
* 60d93181cff3e0b66356283d996d8bab16880686
* 9d1bc0d62fdc11331ecfa420a9001454fd1b8c08
* 16bc176f56fdea784cbfb661920cbc42df371763
* 6825ff4bd388187914768b0c507128dda9830fde
hope to have caught them all, many cooks working on their specific ocds ;p

**Comment by nigoroll** [comment]
re last comment, I appreciated the other contributions and fixes, no bad feelings, it was meant to be a joke

**Comment by Dridi** [comment]
779c46f (the coverity one)

**Comment by nigoroll** [comment]
thx dridi

**Comment by hermunn** [comment]
Backport review: Backported in #2023.

**Pull request closed without merging**